File: sdk/kotlin-spring/src/main/kotlin/org/openapitools/model/NotFound.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param message 
 * @param status 
 */
data class NotFound (

        @JsonProperty("message") val message: String? = null,

        @JsonProperty("status") val status: Int? = null
) {

}

File: sdk/scala-gatling/src/gatling/scala/org/openapitools/client/model/NotFound.scala

package org.openapitools.client.model


case class NotFound (
    _message: Option[String],
    _status: Option[Integer]
)
object NotFound {
    def toStringBody(var_message: Object, var_status: Object) =
        s"""
        | {
        | "message":$var_message,"status":$var_status
        | }
        """.stripMargin
}
